cloudevents/generic/types/types.go

```diff
@@ -48,6 +48,10 @@ const (
 	// ExtensionResourceVersion is the cloud event extension key of the resource version.
 	ExtensionResourceVersion = "resourceversion"
 
+	// ExtensionSequenceID is the cloud event extension key of the event sequence ID.
+	// The event sequence id represents the order in which events occur on the agent/source.
```

Comment:
not the strict order. I think you can only use it to order the event from one single agent. I do not think you need this for source.

Reply:
yeah, the source does not need this, it already has resoruceversion, update the description and variable name to make this more clear